### DISCUSSION TITLE
Releases schedule

### DISCUSSION THREAD

**Discussion by JosePizarro3** [FAIRmat-NFDI/nomad-simulations#96] [Announcements]
@JFRudzinski @ndaelman-hu @Bernadette-Mohr 

I think is a good moment to set up some schedules when coming to releasing a new version of this package.

Being that we are constantly developing, I think we could aim to release a new version every 2 weeks. Please, note that "new version" is just a loose way of specifying a new release into PyPI; it might be as small as a patch (from 0.0.1 to 0.0.2) or as big as it needs to be.

We just need a way of bumping the version in order to be able to connect with nomad-lab in a more robust way.

**Comment by JosePizarro3** [comment]
The next release will be thus on ~09.07.2024~ 11.07.2024. I will add these into our common calendar.

**Reply by ndaelman-hu** [reply]
So, any dependent parser should not rely on the PyPi installation by the tutorial, given that we might not get around to changing the installation setup by then. We can keep it for after, no prob.

What is your recommended way of loading the schema till then?

**Reply by JosePizarro3** [reply]
Not sure if I understand. Targetting 0.0.1 will be safer I think.

But this is an issue which is responsability of Area D to solve and have under control.

**Reply by ndaelman-hu** [reply]
> But this is an issue which is responsability of Area D to solve and have under control.

Exactly, and can we guarantee that if we encounter an issue there the day before the tutorial, that they'll be able to fix it in time?

Releasing a v0.0.1 by itself isn't a problem. I'm just guessing your objective for the date:

- If it's for us to use in the examples, a day before might fall short to update the dependency, given that we will have other things on our minds.
- If it's for the audience to check out and use, I'd say that it isn't mature enough yet. I assume you agree, since we aren't releasing a v0.1.0.

**Reply by JosePizarro3** [reply]
Ah damn, I didn't realize on the actual date, which is one day before the tutorial.

Indeed, we release 0.0.2 on 11.07.2024 better. I'll edit the comment.

**Comment by JosePizarro3** [comment]
@JFRudzinski @Bernadette-Mohr @ndaelman-hu I will be releasing a new version this Thu, 0.0.3 with the new energies.

**Comment by JosePizarro3** [comment]
Next release should happen soon, after @ndaelman-hu implements the `BasisSet`.
